scripts/build_depends_project_list.sh

```
@@ -0,0 +1,41 @@
+  CAT_PATH="$(which cat)"
+fi
+
+function git_root() {
+  git rev-parse --show-toplevel
+}
+
+function list_projects_with_depends_file() {
+  PROJECTS_FOLDER="$(git_root)/$PROJECT_ROOT"
+  for f in $($FIND_PATH $PROJECTS_FOLDER -name .depends)
+  do
+    echo $(basename $(dirname $f))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)